### PR TITLE
ar71xx: Add support for TL-WR740N v.6 with correct LEDs indication

### PR DESCRIPTION
**Pull request by androld**
Signed-off-by: Andrey Polischuk <androld.b@gmail.com>

### CODE REVIEW

**Comment by pepe2k** [comment]
Hi @androld,

Thank you for your first-time contribution!

I have just one request here: instead of adding another mach-*.c file, please include separate definitions for LEDs in the old one (mach-tl-wr841n-v9). Exactly in the same way as it's already done there for TL-WR842N v3, TL-WR841N v11, etc.

Cheers,
Piotr

**Comment by androld** [comment]
Hi @pepe2k,

Thank You for your comment!

I will try to release your request.
I'm not familiar to GIT. Now I will reset commit in my local branch, will make nesessary changes, will test it on hardware and will commit it again. And after it will I need to make new push request? Or I can will update this push request?

Yours sincerely,
Andrey

P.S. Sorry for my English

**Comment by pepe2k** [comment]
Hi @androld,

> I'm not familiar to GIT. Now I will reset commit in my local branch, will make nesessary changes, will test it on hardware and will commit it again. And after it will I need to make new push request? Or I can will update this push request?

Please make all requested changes in your local repository clone and then use `git commit --amend` to add them to your original commit. Use `git push --force` to overwrite your GitHub repository/branch, which will afterwards update your pull request here automatically.

Cheers,
Piotr

**Comment by androld** [comment]
Did it!
@pepe2k please check - All correct I doing?

File: target/linux/ar71xx/files/arch/mips/ath79/Makefile [hunk 1]
```diff
@@ -198,6 +198,7 @@ obj-$(CONFIG_ATH79_MACH_TL_WR1043ND_V4)		+= mach-tl-wr1043nd-v4.o
 obj-$(CONFIG_ATH79_MACH_TL_WR2543N)		+= mach-tl-wr2543n.o
 obj-$(CONFIG_ATH79_MACH_TL_WR703N)		+= mach-tl-wr703n.o
 obj-$(CONFIG_ATH79_MACH_TL_WR720N_V3)		+= mach-tl-wr720n-v3.o
+obj-$(CONFIG_ATH79_MACH_TL_WR740N_V6)		+= mach-tl-wr740n-v6.o
```
Comment (androld) [inline target/linux/ar71xx/files/arch/mips/ath79/Makefile]: Yes, I compile it and test on router - all works fine.
But if this config wrong - I will fix it, compile and test again.
Thanks!

**Comment by androld** [comment]
Fixed

**Comment by pepe2k** [comment]
@androld, thank you! I have merged your changes with minor fixes into [my staging tree](https://git.lede-project.org/?p=lede/pepe2k/staging.git;a=summary).

**Comment by androld** [comment]
@pepe2k 
Thank you!

**Pull request closed without merging**